presidio-analyzer/presidio_analyzer/app_tracer.py

```diff
@@ -12,9 +12,7 @@ class AppTracer:
     """
 
     def __init__(self, enabled: bool = True):
-
-        self.logger = PresidioLogger("Interpretability")
-        self.logger.set_level("INFO")
+        self.logger = logging.getLogger("decision_process")
```

Comment:
should the rename be in the api as well? remove_interpretability_response attribute?

Reply:
Yes, we have a user story for this.